ci-operator/step-registry/upi/conf/aws/proxy/upi-conf-aws-proxy-ref.yaml

```diff
@@ -1,6 +1,9 @@
 ref:
   as: upi-conf-aws-proxy
-  from: upi-installer
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
```

Comment:
The 4.12 upi-installer will still be the rhel7-based one, right? At least until we backport the changes.

Reply:
yes, currently still use rhel7-based 4.12 upi-installer. But should be no impact, since yq-go exists on both rhel7-based and rhel8-based upi-installer.

Reply:
Fair enough.